pywal/colors.py

```diff
@@ -85,11 +84,10 @@ def get(img, cache_dir=CACHE_DIR,
         color_count=COLOR_COUNT, notify=False):
     """Get the colorscheme."""
     # _home_dylan_img_jpg.json
-    cache_file = cache_dir / "schemes" / \
-        img.replace("/", "_").replace(".", "_")
-    cache_file = cache_file.with_suffix(".json")
+    cache_file = os.path.join(cache_dir, "schemes",
+                              img.replace("/", "_"), ".json")
```

Comment:
this creates a folder with a file named ".json" inside which is slightly different from previous behaviour, this could be solved by adding `+` instead of the last comma.